src/main/java/frc/robot/commands/SwerveDriveCommands/DriveConstantControlCommand.java

```diff
@@ -1,98 +1,105 @@
 package frc.robot.commands.SwerveDriveCommands;
 
+import static frc.robot.Constants.DriveConstants.maxAngularSpeed;
+import static frc.robot.Constants.DriveConstants.maxSpeed;
+
 import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import frc.robot.subsystems.swerve.DetermineDriveSpeed;
+import frc.robot.subsystems.swerve.DriveSpeed;
 import frc.robot.subsystems.swerve.SwerveDrive;
-
-import static frc.robot.Constants.AutoConstants;
-import static frc.robot.Constants.OIConstants;
+//import frc.robot.subsystems.swerve.SwerveModule;
+import frc.robot.Constants.*;
 
 public class DriveConstantControlCommand extends CommandBase {
 
-  private final SwerveDrive m_swerveDrive;
-  private final XboxController controller;
-  private final DetermineDriveSpeed limJoystickLeft = new DetermineDriveSpeed(0.05);
-  private final DetermineDriveSpeed limJoystickRight = new DetermineDriveSpeed(0.05);
+    private final SwerveDrive m_swerveDrive;
+    private XboxController controller;
+    private final DriveSpeed limJoystickLeft = new DriveSpeed(0.05);
+    private final DriveSpeed limJoystickRight = new DriveSpeed(0.05);
+    public DriveConstantControlCommand(SwerveDrive swerveDrive, XboxController controller) {
+        m_swerveDrive = swerveDrive;
+        this.controller = controller;
+        addRequirements(swerveDrive);
+    }
 
-  public DriveConstantControlCommand(SwerveDrive swerveDrive, XboxController controller) {
-    m_swerveDrive = swerveDrive;
-    this.controller = controller;
 
-    addRequirements(swerveDrive);
-  }
 
-  @Override
-  public void initialize() {}
+    @Override
```

Comment:
Why is the indentation in this file so inconsistent? VS Code should be able to reformat the whole file automatically so it's correct.